boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Jefferson Lee
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+    model = "Arduino Nano 33 BLE (Sense)";
+    compatible = "arduino,arduino_nano_33_ble";
+
+    chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &code_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 24 0>;
+			label = "Red LED";
+		};
+	};
+
```

Comment:
On the Nano 33 BLE I have, red is pin 22, green 23 and blue 24 for the RGB LED. The power LED is 25 and the Arduino "built in" LED is 13.